backend/tournesol/views/ratings.py

```diff
@@ -54,9 +58,21 @@ def get_annotated_ratings():
         .order_by("-datetime_lastedit")
     )[:1]
 
+    collective_score = (
+        EntityPollRating.objects.filter(poll=OuterRef("poll"), entity=OuterRef("entity"))
+        .values("tournesol_score")
+    )
+
+    contributor_rating_criteria_score = (
+        ContributorRatingCriteriaScore.objects.filter(contributor_rating=OuterRef("pk"), criteria="largely_recommended")
```

Comment:
This should be replaced `poll.main_criteria` such that it works for other polls too.

Reply:
Thank you for this clarification, it is changed.